app/validation.py

```diff
@@ -115,6 +125,22 @@ def validates_against_schema(validator_name, submitted_json):
         return True
 
 
+def get_validation_errors(validator_name, json_data, enforce_required=True):
+    error_map = {}
+    try:
+        validator = get_validator(validator_name, enforce_required)
+        errors = sorted(validator.iter_errors(json_data), key=lambda e: e.path)
+        for index, error in enumerate(errors):
+            if error.path:
+                key = error.path.pop()
+            else:
+                key = "err_{}".format(index)
```

Comment:
How are we going to tie the errors to individual fields for display? Is it possible to do something like this instead:
```json
{
  "errors": [
     "Request isn't even JSON"
  ],
  "field_errors": {
    "serviceName": "Bad name",
    "pricingDocumentURL": "Not a URL"
}
```

If we can get more than one error per field from the jsonschema, then I'd change that to `"field": ["error 1", "error 2"], "other_field": ["single error"]`.

Alternatively, instead of splitting "errors" and "field_errors" we could have a special key for form errors (eg "_form").